package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "todo_app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) **in our case we are missing `webpack-dev-server`**
Kindly add `webpack-dev-server`.

We also need to inform users about these scripts in the README file. This information can be placed in the Usage section. This way users can know the command to run to start your application.

Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example;
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

 I think we could deleted the test script since we have no tests implemented.

Reply:
Thanks for your feedback. "getting prefilled form" is done on purpose.